packages/eas-cli/src/commands/worker/deploy.ts

```diff
@@ -36,6 +42,9 @@ export default class WorkerDeploy extends EasCommand {
   static override state = 'beta';
 
   static override flags = {
+    prod: Flags.boolean({
+      description: 'Deploy to production',
+    }),
```

Comment:
nit: if you want to you can even do
```suggestion
    prod: Flags.boolean({
      description: 'Deploy to production',
      default: false,
    }),
```
to make flag required, but with default value

Reply:
Oh I see, to remove the sanitization part, right?